client/src/__tests__/components/grants/List.test.tsx

```diff
@@ -143,7 +143,7 @@ describe("<List />", () => {
 
         renderWrapped(<List />, store);
 
-        expect(screen.getByText("loading...")).toBeInTheDocument();
+        expect(screen.findByDisplayValue("Loading...")).toBeTruthy();
```

Comment:
@codenamejason `findBy...` queries return a `Promise` so it's always truthy if you don't use `await`.
In this case `getByText` fails because it finds more than 1 element, cause the `Spinner` component already has a `Loading...` label for accessibility. We can use `getAllByText` and check that they are 2, or set the `label` attribute of the `Spinner` component to something like `Loading Projects...` and `Loading Round...` so that they are different.